src/screens/cars/AddCar.tsx

```diff
@@ -0,0 +1,374 @@
+                defaultValue={selectedConverter}
+                defaultButtonText={`${I18n.t('cars.converter')}*`}
+                data={selectedCarCatalogConverters}
+                buttonTextAfterSelection={(carConverter: CarConverter) => Utils.buildCarCatalogConverterName(carConverter)}
+                rowTextForSelection={(carConverter: CarConverter) => Utils.buildCarCatalogConverterName(carConverter)}
+                buttonStyle={[style.selectField, !selectedCarCatalog && style.selectFieldDisabled]}
+                buttonTextStyle={style.selectFieldText}
+                dropdownStyle={style.selectDropdown}
+                rowStyle={style.selectDropdownRow}
+                rowTextStyle={style.selectDropdownRowText}
+                renderDropdownIcon={() => <Icon type={'MaterialIcons'} name={'arrow-drop-down'} />}
+                onSelect={(carConverter: CarConverter) => this.setState({ selectedConverter: carConverter })}
+              />
+            )}
+          />
+          <Input
+            placeholder={Constants.VIN_FULL}
+            placeholderTextColor={commonColors.disabledDark}
+            labelStyle={style.inputLabel}
+            autoCapitalize={'none'}
+            autoCorrect={false}
+            label={`${Constants.VIN}*`}
+            errorMessage={!this.checkVIN() && vin && I18n.t('cars.invalidVIN')}
+            errorStyle={style.errorText}
+            inputStyle={style.selectDropdownRowText}
+            onChangeText={(newVin: string) => this.setState({ vin: newVin })}
+          />
+          <Input
+            placeholder={I18n.t('cars.licensePlate')}
+            placeholderTextColor={commonColors.disabledDark}
+            label={`${I18n.t('cars.licensePlate')}*`}
+            labelStyle={style.inputLabel}
+            autoCapitalize={'none'}
+            autoCorrect={false}
+            errorStyle={style.errorText}
+            errorMessage={!this.checkLicensePlate() && licensePlate && I18n.t('cars.invalidLicensePlate')}
+            inputStyle={style.selectDropdownRowText}
+            onChangeText={(newLicensePlate: string) => this.setState({ licensePlate: newLicensePlate })}
+          />
+          {this.securityProvider?.canListUsers() && (
+            <ModalSelect<User>
+              openable={true}
+              disabled={false}
+              defaultItem={selectedUser}
+              renderItem={(user: User) => <UserComponent shadowed={true} user={user} navigation={navigation} />}
+              onItemsSelected={(users: User[]) => this.setState({ selectedUser: users?.[0] })}
+              navigation={navigation}
+              selectionMode={ItemSelectionMode.SINGLE}>
+              <Users navigation={navigation} />
+            </ModalSelect>
+          )}
+          <TouchableOpacity onPress={() => this.setState({ isDefault: !isDefault })} style={style.defaultContainer}>
+            <CheckBox style={formStyles.checkbox} checked={isDefault} disabled={true} />
```

Comment:
Same proportions between checkbox and radio button.
You can also use the Switch control.

<img width="453" alt="2021-10-04_10-38-24" src="https://user-images.githubusercontent.com/25266945/135820465-c26a8433-e03e-40e3-a6a9-d15b4773c919.png">